### DISCUSSION TITLE
Astro collection integration

### DISCUSSION THREAD

**Discussion by florian-lefebvre** [Thinkmill/keystatic#313] [Q&A]
Hi there! I'm planning to integrate Keystatic to an existing Astro app that uses collections (from v2). How does Keystatic integrates with it?

I think it could be useful to talk about this in the docs as well

**Comment by simonswiss** [comment]
Hey Florian!

You're absolutely right — this is something we need to add to the docs.

The good news is Keystatic works **really well** with Astro collections. You can skip the whole Keystatic Reader API and use the content collections instead. 

And it keeps getting better as Astro adds new features like:

- support for YAML and JSON formats in collections
- hybrid rendering with "opt-out" of static render

We'll definitely add docs for this, but in the meantime you may find this video I made a few months ago somewhat useful:

[![Watch the video](https://img.youtube.com/vi/6l2YWCyPsWk/maxresdefault.jpg)](https://youtu.be/6l2YWCyPsWk)

**Reply by florian-lefebvre** [reply]
Awesome! I'll check it out and let you know if I have any question about it 👍

**Reply by simonswiss** [reply]
Great!
